src/VehicleSetup/JoystickConfig.qml

```diff
@@ -87,6 +87,20 @@ SetupPage {
                         color:                  __barColor
                     }
 
+                    // Deadband
```

Comment:
By default deadbands will be disabled as such there should be no difference to users. It is only when they active them.